packages/material/scss/autocomplete/_layout.scss

```diff
@@ -1 +1,7 @@
 @import "~@progress/kendo-theme-default/scss/autocomplete/layout";
+
+@include exports('autocomplete/layout/material') {
+    .k-autocomplete {
+        border-radius: 0;
```

Comment:
Introduce a `$autocomplete-border-radius` in theme-default and use it to reset it to zero here.

Reply:
the variable is added